tests/data_context/test_data_context.py

```diff
@@ -72,35 +72,6 @@ def data_context_with_bad_datasource(tmp_path_factory):
     return get_context(context_root_dir=context_path)
 
 
-@pytest.mark.cloud
-def test_save_expectation_suite_include_rendered_content(
```

Comment:
Looks like I had ported this test over, but the intention of the test was to test context CRUD methods, so removing it.